libs/designsystem/slide/src/slides.component.scss

```diff
@@ -70,6 +72,11 @@ $bullet-inactive-opacity: 0.2;
     display: none;
   }
 
+  swiper-container {
+    padding: utils.size('m') utils.size('s');
+    margin: calc(-1 * utils.size('m')) calc(-1 * utils.size('s'));
```

Comment:
Consider simplifying as calculations are built into Sass — and you can do just negate the values:
```suggestion
    margin: -utils.size('m') -utils.size('s');
```

Reply:
That's nice to know!